services/meta/client.go

```diff
@@ -0,0 +1,119 @@
+package meta
```

Comment:
This is just the mocked out client.  We need to evaluate some of the behavior and if other services should use them the way the did before.  Most notable are things like `WaitForLeader`, `IsLeader`, `WaitForDataChanged` etc.

It's less about do we need that functionality, and more about how to re-implement that functionality to be more understandable and durable with the new Client/Service architecture.